examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -59,6 +60,12 @@ AllClustersCommandDelegate sAllClustersCommandDelegate;
 Clusters::WindowCovering::WindowCoveringManager sWindowCoveringManager;
 
 Clusters::TemperatureControl::AppSupportedTemperatureLevelsDelegate sAppSupportedTemperatureLevelsDelegate;
+
+// Please refer to https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/namespaces
+const Clusters::Descriptor::Structs::SemanticTagStruct::Type gTagList[] = {
+    { .namespaceID = 7, .tag = 0 }, // Common Number Namespace: 7, tag 0 (Zero)
+    { .namespaceID = 8, .tag = 2 }
```

Comment:
There should be constant names (even if just declared here in anonymous namespace) to replace these magic numbers.